### ISSUE TITLE
after iframe2 is merged into master, set up django server to autodeploy from master to `extremely-alpha.iodide.io`

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#749]
blocked by  #747
cc @mdboom 

once iframe is merged into master, we want to get the live  server up and running with the iframe. this will require a couple webpack tweaks as mentioned in the meeting

also @wlach, lmk what we level of heroku account we should have. i'll talk to matt_g and get us the upgrade.

**Comment by wlach** [comment]
re: heroku accounts I think we can live with the free dyno for now, it doesn't take *that* long for the instance to spin up.

**Comment by wlach** [comment]
I think we fixed this one!
